test_team3.py

```diff
@@ -16,3 +16,39 @@ def test_anonymize_nrp(self):
 
         self.assertEqual('My NRP is Unknown',
                          anonymize_text('My NRP is Unknown', ['NRP']))
+       
+    def test_credit_card(self):
+        """Test to make sure CREDIT_CARD is recognized"""
+        #Positive Test Case 
+        sec1 = '5105'
+        sec2 = '1051'
+        sec3 = '0510'
+        sec4 = '5100'
+        test_cc = sec1 + sec2 + sec3 + sec4
+        test_string = 'My credit card number is ' + test_cc
+        expected_result = 'My credit card number is <CREDIT_CARD>'
+        actual_result = anonymize_text(test_string, ['CREDIT_CARD'])
+        self.assertEqual(expected_result,actual_result) 
+
+        #Negative Test Case - Credit Card Number invalid
+        test_cc = '1232 8324 3847 3488' 
+        test_string = 'My credit card number is ' + test_cc
+        expected_result = 'My credit card number is 1232 8324 3847 3488'
+        actual_result = anonymize_text(test_string, ['CREDIT_CARD'])
+        self.assertEqual(expected_result,actual_result)
+
+        #Negative Test Case - Credit Card Number too short
+        test_cc = '1232 8324' 
+        test_string = 'My credit card number is ' + test_cc
+        expected_result = 'My credit card number is 1232 8324'
+        actual_result = anonymize_text(test_string, ['CREDIT_CARD'])
+        self.assertEqual(expected_result,actual_result)
+
+        #Negative Test Case - Credit Card Number too long: beyond 19 digits
+        test_cc = '1232 8324 3453 2353 6575 2344' 
+        test_string = 'My credit card number is ' + test_cc
+        expected_result = 'My credit card number is 1232 8324 3453 2353 6575 2344'
+        actual_result = anonymize_text(test_string, ['CREDIT_CARD'])
+        self.assertEqual(expected_result,actual_result)
+
```

Comment:
Great , What about a negative test case for trying to enter a non numeric character ?

Reply:
Change have been made. Thank you for the comment.